frontend/src/metabase/actions/components/ActionViz/ActionOptions.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { useCallback, useMemo } from "react";
+import { t } from "ttag";
+import { connect } from "react-redux";
+import _ from "underscore";
+
+import { useToggle } from "metabase/hooks/use-toggle";
+
+import Icon from "metabase/components/Icon";
+import Select from "metabase/core/components/Select";
+import Button from "metabase/core/components/Button";
+
+import TippyPopoverWithTrigger from "metabase/components/PopoverWithTrigger/TippyPopoverWithTrigger";
+import ActionCreator from "metabase/actions/containers/ActionCreator";
+
+import Actions from "metabase/entities/actions";
+import Questions from "metabase/entities/questions";
+import { setParameterMapping } from "metabase/dashboard/actions";
+
+import type {
+  ActionDashboardCard,
+  ActionParametersMapping,
+  WritebackParameter,
+  WritebackAction,
+  Dashboard,
+} from "metabase-types/api";
+
+import type { State } from "metabase-types/store";
+import type {
+  ParameterTarget,
+  ParameterId,
+} from "metabase-types/types/Parameter";
+import type Question from "metabase-lib/Question";
+
+import {
+  ActionParameterTriggerContainer,
+  ParameterMapperContainer,
+  ParameterFormSection,
+  ParameterFormLabel,
+  ParameterMapperTitleContainter,
```

Comment:
Typo: should be `Container` instead of `Containter` (`ParameterMapperTitleContainer`)